src/sentry/static/sentry/app/views/organizationIntegrations/providerRow.tsx

```diff
@@ -114,25 +112,41 @@ export default class ProviderRow extends React.Component<Props> {
   render() {
     return (
       <PanelItem p={0} direction="column" data-test-id={this.props.provider.key}>
-        <Flex align="center" p={2}>
+        <PanelItemFlex>
           <PluginIcon size={36} pluginId={this.props.provider.key} />
-          <Box px={2} flex={1}>
+          <ProviderBox>
             <ProviderName>{this.props.provider.name}</ProviderName>
             <ProviderDetails>
               <Status enabled={this.isEnabled} />
               <StyledLink onClick={this.openModal}>Learn More</StyledLink>
             </ProviderDetails>
-          </Box>
+          </ProviderBox>
           <Box>
             <Button size="small" onClick={this.openModal} {...this.buttonProps} />
           </Box>
-        </Flex>
+        </PanelItemFlex>
         {this.renderIntegrations()}
       </PanelItem>
     );
   }
 }
 
+const Flex = styled('div')`
+  display: flex;
+`;
+const Box = styled('div')``;
```

Comment:
@NisanthanNanthakumar If there is no styling at all with this component, we can use a plain div.